File: problems/f/faktor/faktor.c
/*
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
*/

#include <stdio.h>

int main(void) {
    int a, i, scientists;
    if (scanf("%d%d", &a, &i) != 2) {
        printf("Failed to read a and i!");
    }

    scientists = a * (i - 1) + 1;

    printf("%d", scientists);

    return 0;
}

File: problems/f/faktor/faktor.py
#! /usr/bin/python3
"""
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
"""

def main():
    a, i = [int(val) for val in input().split()]
    scientists = a * (i - 1) +1
    print(scientists)

if __name__ == '__main__':
    main()